### DISCUSSION TITLE
Autoavvio Home Assistant Core su MiniPC

### DISCUSSION THREAD

**Discussion by pigmos** [indomus/forum#2619] [HUB: Home Assistant]
Buongiorno,
ho sgeuito la guida per installare Home Assistant Core su MiniPC, con sistema operativo di base Debian 12, Docker e Home Assistant COre istanziato mediante Docker Compose. E tutto funziona senza problemi.
La cosa che non capisco, o credo di aver capito, è che ora Docker e Home Assistant devono essere avviati. Io vorrei creare un server che anche in caso di reboot (per qualsiasi motivo, è dietro UPS, ma potrebbe capitare qualsiasi cosa) faccia ripartire i servizi. Nella guida questo non è indicato, poichè sono poco esperto di Linux (e di Debian in particolare) qualcuno può dirmi se ho cpaito bene, e come realizzare questa automazione.
Grazie.

**Comment by KC-inDomus** [comment]
scusa, prova a riavviare il mini pc.
non riparte tutto automaticamente?

**Reply by snakuzzo** [reply]
docker, appena installato, è già configurato per partire in automatico.
Basta controllare con

```
sudo systemctl is-enabled docker.service
```

Per quanto riguarda invece i servizi definiti nel compose, se hai impostato `restart: always` partirà in automatico dopo lo start del servizio docker

**Reply by pigmos** [reply]
Grazie Snakuzzo, è esattamente come dici, docker parte come sistema in avvio, ed i servizi sono autovvianti con docker. Non ci avevo fatto caso. Scusatemi, ma sono novello di debian e mi sono fatto prendere dall'ansia.
